common/logging.go

```diff
@@ -60,3 +60,20 @@ func InitDefaultLogging(debug bool) {
 	}
 	InitLogging(level)
 }
+
+func SetLogLevel(levelname string) error {
+	level, err := logrus.ParseLevel(levelname)
+	if err != nil {
+		return err
+	}
+	InitLogging(level)
+	return nil
+}
+
+// Combination of InitDefaultLogging and SetLogLevel, for convenience
+// of existing programs that support both options
+func InitDefaultLoggingOrDie(loglevel string) {
+	if err := SetLogLevel(loglevel); err != nil {
+		Log.Fatal(err)
+	}
+}
```

Comment:
It was to minimize impact on unrelated code.  But I think four init functions is taking this too far; I will clean it up.